### DISCUSSION TITLE
Unable to login

### DISCUSSION THREAD

**Discussion by ChrisKoh83** [nVuln/homebridge-lg-thinq#68] [General]
Hey guys,

I´m trying to configure the plugin but I´m still getting an error message:

![image](https://user-images.githubusercontent.com/13451391/131839500-06556b00-30a5-4291-bc8b-5a4f5e8ae051.png)

![image](https://user-images.githubusercontent.com/13451391/131839420-5bf55322-117a-499e-9e27-6acff57886d9.png)
 
If I go to lg.com I´m able to login....

![image](https://user-images.githubusercontent.com/13451391/131840102-5291cb5d-cbb2-4403-8975-9cd11af287be.png)

![image](https://user-images.githubusercontent.com/13451391/131840209-03a8e9e6-e67f-4add-a82d-1f05f26e1aed.png)

**Comment by nVuln** [comment]
did you try login to native LG ThinQ app ?

**Reply by ChrisKoh83** [reply]
Yes:

![image](https://user-images.githubusercontent.com/13451391/131846365-5227dfdc-36d4-469a-902a-90e615f02979.jpeg)

**Reply by nVuln** [reply]
Not sure what happened on your account, I try created new account on Germany and it's still worked for me, nothing wrong
Please try create new account

**Reply by ChrisKoh83** [reply]
Thanks, I´ll try it!

**Reply by malwcal** [reply]
Also to double check: Did you sign out of the app and sign in again?  I needed to do that to accept the new terms and conditions.  Before that I was using the app since last year but it just stayed logged in.

**Comment by ChrisKoh83** [comment]
Now it's working!
 There were several passwords for my LG account were saved in my keychain.
Thanks to the logout and attempt to login in again , I noticed that!
I did a cleanup and recovered the password for my existing account and TA TA .... it works fine!

Thank you for your support!
